dev/report.json

```diff
@@ -0,0 +1,18 @@
+{
+	"reports": [
+		{
+			"date": "2018-12-11T16:36:00.500Z",
+			"git": "5add401dcf795856ecbcac60c27234cffb90cc78",
```

Comment:
this commit does not exist anymore on this branch.  
Please run the bench on the new head